settings.gradle.kts

```diff
@@ -5,7 +5,6 @@
 
 pluginManagement {
     repositories {
-        maven("https://dl.bintray.com/kotlin/kotlin-eap")
```

Comment:
**Question**: What was this for? Why can we now remove it?

Reply:
I don't recall what it was for, it's from very early on. All references to `jcenter` and/or `bintray` should be removed though because they are [shutting down](https://blog.gradle.org/jcenter-shutdown). I actually just recently removed `jcenter` from this repo but missed this reference. Happened to catch it debugging something else. Having it present slows our builds down because it tries to resolve plugins/dependencies and gets an error response and then resolves using a different repo.